src/mumble/UserLocalVolumeSlider.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "UserLocalVolumeSlider.h"
+#include "ClientUser.h"
+
+#include <cmath>
+
+
+
+
+UserLocalVolumeSlider::UserLocalVolumeSlider(QWidget *parent) 
+	: VolumeSliderWidgetAction(parent) {}
+
+void UserLocalVolumeSlider::prepareLocalUser(unsigned int sessionId) {
+
+	m_clientSession = sessionId;
+
+	ClientUser *user = ClientUser::get(sessionId);
+
+	if (user) {
+		updateSliderValue(user->getLocalVolumeAdjustments());
+	}
+}
```

Comment:
I can pass directly the `ClientUser` as reference to avoid `nulptr` and unnecessary lookups and keep a pointer to it. 

Since I will want to change the `ClientUser` volume I should not pass it as `const` correct?
`ClientUser::setLocalVolumeAdjustment` is not const.